Documentation/articles/getting_started/1_setting_up_your_development_environment_ubuntu.md

```diff
@@ -0,0 +1,87 @@
+# Setting up your development environment for Ubuntu 20.04
+
+In this section we will go over setting up your development environment for Ubuntu 20.04.
+
+## Install .NET Core SDK:
+
+Add repository:
+```sh
+wget https://packages.microsoft.com/config/ubuntu/20.04/packages-microsoft-prod.deb -O /tmp/packages-microsoft-prod.deb
+sudo dpkg -i /tmp/packages-microsoft-prod.deb
+sudo apt update
+```
+
+Install packages:
+```sh
+sudo apt-get install -y apt-transport-https
+sudo apt-get install -y dotnet-sdk-3.1
+```
+
+## [Optional] Install mono:
+
+Mono is a C# runtime, just like .NET Core, and it is not nessesary to install it if you are just targeting Linux, however if you want to target some other platforms, like Android, it will be required.
+
+Add repository:
+```sh
+sudo apt install gnupg ca-certificates
+sudo apt-key adv --keyserver hkp://keyserver.ubuntu.com:80 --recv-keys 3FA7E0328081BFF6A14DA29AA6A19B38D3D831EF
+echo "deb https://download.mono-project.com/repo/ubuntu stable-focal main" | sudo tee /etc/apt/sources.list.d/mono-official-stable.list
+sudo apt update
+```
+
+Install packages:
+```sh
+sudo apt install -y mono-devel
+```
+
+## Install Visual Studio Code
+
+Add repository:
+```sh
+curl https://packages.microsoft.com/keys/microsoft.asc | gpg --dearmor > /tmp/packages.microsoft.gpg
+sudo install -o root -g root -m 644 /tmp/packages.microsoft.gpg /etc/apt/trusted.gpg.d/
+sudo sh -c 'echo "deb [arch=amd64 signed-by=/etc/apt/trusted.gpg.d/packages.microsoft.gpg] https://packages.microsoft.com/repos/vscode stable main" > /etc/apt/sources.list.d/vscode.list'
+sudo apt update
+```
+
+Install packages:
+```sh
+sudo apt-get install code
+```
+
+Install C# extension:
+```
+code --install-extension ms-dotnettools.csharp
+```
+
+## Install MonoGame templates
+
+This will install templates for .NET CLI tools and Rider IDE. There is no template support for MonoDevelop.
+
+```sh
+dotnet new --install MonoGame.Templates.CSharp
+```
+
+## Install MGCB Editor
```

Comment:
Like the other guides, this should add a reference to the MGCB-Editor page to avoid duplicating the instructions which are the same for all systems.